spec/openapi.yaml

```diff
@@ -675,6 +720,26 @@ components:
             type: string
           example:
             ADD_SUMMARY_TO_NARRATIVE: true
+    HealthCheckFailureNotify:
+      type: object
+      properties:
+        errorType:
+          type: string
+          description: >
+            Type of non-transient health check failure that occurred.
+          enum:
+           - UNKNOWN
+           - TARGET_UNREACHABLE
+           - TARGET_AUTH_FAILURE
+        errorKey:
```

Comment:
I have left `errorKey` here – what do you think of using `errorType` enum that we expand over time?
It might make messaging to end users more helpful to team and administrators if scope is restricted.

Reply:
I left it open intentionally only mentionioning UNKNOWN to be used by users of the API other than us, which then would require to add the error message.
But looking at your suggestion it's actually better to use an expanding enum.
I'll update the PR